packages/volto/locales/volto.pot

```diff
@@ -2355,6 +2356,11 @@ msgstr ""
 msgid "Navigate back"
 msgstr ""
 
+#. Default: "Navigate to this item"
+#: components/manage/Contents/ContentsDeleteModal
+msgid "Navigate to this items"
```

Comment:
This should be either "Navigate to this item" or the "Navigate to these items", and align with the Default. I'm not sure whether it should be plural or singular. Please advise.